docs/en/tutorials/how_to_install_mmdeploy_on_jetsons.md

```diff
@@ -26,98 +26,164 @@ Sometimes we just need to reboot the jetson device when it gets stuck in initial
 ### Cuda
 
 The Cuda is installed by default while the cudnn is not if we use the first method. We have to write the cuda path and lib to `$PATH` and `$LD_LIBRARY_PATH`:
-```
+```shell
 export PATH=$PATH:/usr/local/cuda/bin
 export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/local/cuda/lib64
 ```
 Then we can use `nvcc -V` the get the version of cuda we use.
 
-### Anaconda
+### Conda
 
 We have to install [Archiconda](https://github.com/Archiconda/build-tools/releases) instead as the Anaconda does not provide the wheel built for jetson.
+```shell
+wget https://github.com/Archiconda/build-tools/releases/download/0.2.3/Archiconda3-0.2.3-Linux-aarch64.sh
+bash Archiconda3-0.2.3-Linux-aarch64.sh
+source ~/.bashrc
+conda --version
+```
```

Comment:
may point out this is an example. Not a restricted version requirement.

Reply:
Done